.bazeliskrc

```diff
@@ -1,2 +1,2 @@
 BAZELISK_BASE_URL=https://github.com/aspect-build/aspect-cli/releases/download
-USE_BAZEL_VERSION=aspect/5.8.19
+USE_BAZEL_VERSION=aspect/5.9.25
```

Comment:
Should we do some of these in 2.x?

Reply:
We could but I'd rather optimize for time here since there are 9 of these downstream repos and if we start pre-factoring that is dozens more PRs to create and manage. I miss monorepo.

Reply:
Ugh. The bazel 7 upgrade snowballed and now it is half of this PR. I'll pre-factor that to main.

Reply:
https://github.com/aspect-build/rules_ts/pull/598